packages/homepage/src/templates/docs.js

```diff
@@ -274,7 +274,6 @@ export const pageQuery = graphql`
             title
           }
           fields {
-            slug
```

Comment:
Don't we need slug for docs?

Reply:
They don't even have them in the code ahah